libuavcan_drivers/stm32/driver/src/internal.hpp

```diff
@@ -11,7 +11,7 @@
 #elif UAVCAN_STM32_NUTTX
 # include <nuttx/arch.h>
 # include <arch/board/board.h>
-# include <chip/stm32_tim.h>
+#include <stm32_tim.h>
```

Comment:
Why is this change needed here?

Reply:
I think this is a remain of something else. @ARob109 I think you applied your changes over a commit which is not the same HEAD as master. Please do a git rebase first.

Reply:
Sorry about that. Still new to the process, got it rebased and issued new pull request.

Reply:
Thank you. For your future reference, you don't need to open a new PR after every change. You could have just pushed to this branch and GitHub would have updated the current PR automatically.